.github/workflows/build_library.yml

```
@@ -0,0 +1,27 @@
+name: Binary Release Build
+
+on:
+  push:
+    branches:
```

I recommend adding `master` here too to get nightlies. We have nightly builds for all Sentry components.
```suggestion
    branches:
      - master
```